lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -118,12 +118,13 @@ def only_forwards_all?(send_classifications)
           send_classifications.each_value.all? { |c, _, _| c == :all }
         end
 
-        def add_forward_all_offenses(node, send_classifications)
-          send_classifications.each_key do |send_node|
-            register_forward_all_offense_on_forwarding_method(send_node)
+        def add_forward_all_offenses(node, send_classifications, forwardable_args)
+          send_classifications.each do |send_node, (_c, forward_rest, _forward_kwrest)|
```

Comment:
We need to capture the `forward_rest` value so that we can replace only after the splat if leading arguments are allowed e.g. in 
```ruby
def foo(x, *args, **kwargs, &block)
```
vs
```ruby
def foo(x, ...)
```